ospfd/ospf_dump.h

```diff
@@ -42,17 +42,18 @@
 #define OSPF_DEBUG_NSM_STATUS	0x01
 #define OSPF_DEBUG_NSM_EVENTS	0x02
 #define OSPF_DEBUG_NSM_TIMERS   0x04
-#define OSPF_DEBUG_NSM		0x07
+#define OSPF_DEBUG_NSM 0x07
 
 #define OSPF_DEBUG_LSA_GENERATE 0x01
-#define OSPF_DEBUG_LSA_FLOODING	0x02
```

Comment:
Why do we need all these whitespace changes in this file?

Reply:
I think , these white space changes are added due to clang tool . Addressed.